src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -401,8 +532,8 @@ def postprocess(self, model_outputs, decoder_kwargs: Optional[Dict] = None, retu
         # Optional return types
         optional = {}
 
-        if return_timestamps and self.type == "seq2seq":
-            raise ValueError("We cannot return_timestamps yet on non-ctc models !")
+        if return_timestamps and self.type == "seq2seq" and "Whisper" not in self.model.__class__.__name__:
```

Comment:
Given the amount of code that relies on this I would suggest creating `self.type == "seq2seq_whisper"`. 

In general I'm not in favor of model specific code within pipelines, but given the popularity of whisper I'm willing to make an exception (and also the amount of highly specific code).

Also I would switch the condition to something more robust like:

self.model.config.__class__.__name__ == "WhisperConfig"  (This is not as clean as `isinstance(self.model.config, WhisperConfig)` but it avoids importing the model specific class which is preferred sometimes to avoid tangling too directly code.